src/libsyntax/parse/parser/expr.rs

```diff
@@ -1043,13 +1042,12 @@ impl<'a> Parser<'a> {
                         // recovery in order to keep the error count down. Fixing the
                         // delimiters will possibly also fix the bare semicolon found in
                         // expression context. For example, silence the following error:
-                        // ```
```

Comment:
Why? It was good as-is.

Reply:
I see backticks as a bit unnecessary and obtrusive in non-doc comments, compared to indentation. Moreover, indentation definitely seems to be the norm for non-doc-comments.